awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupList.jsx

```diff
@@ -57,18 +61,32 @@ function InstanceGroupList() {
       actions,
       relatedSearchableKeys,
       searchableKeys,
+      defaultControlPlan,
+      defaultExecution,
     },
   } = useRequest(
     useCallback(async () => {
       const params = parseQueryString(QS_CONFIG, location.search);
 
-      const [response, responseActions] = await Promise.all([
+      const [
```

Comment:
@AlexSCorey Will the new settings request run once on initialization or every time the user applies a search filter? If it's running for every search, we'll want to put this into it's own `useRequest` hook and make sure we only call it once.